src/tests/ngraph_helpers/ngraph_functions/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ addIeTarget(
                 openvino::runtime
                 ngraph_reference
                 interpreter_backend
+                inference_engine_transformations
```

Comment:
let's try to replace `inference_engine_transformations` with `openvino::runtime::dev` to fix the CI

Reply:
done.